moped-editor/src/components/GridTable/GridTableNewItem.js

```diff
@@ -13,21 +13,16 @@ import { NavLink as RouterLink } from "react-router-dom";
 const GridTableNewItem = ({ query }) => {
   return (
     <Box display="flex" justifyContent="flex-end">
-      {(query.config.showNewItemButton && query.config.customNewItemButton) ||
```

Comment:
It looks like we built in future support for "new item buttons", however this is only used in the staff listing, and the staff listing uses the `customNewItemButton`. 

in any case, because this render logic was inside the component, it was adding an empty div to the top of the project list.

<img width="1192" alt="Screen Shot 2022-06-16 at 2 05 09 PM" src="https://user-images.githubusercontent.com/14793120/174136839-1bce3fda-ed53-4558-9623-29f6c78600d1.png">

Reply:
the add new project button used to be in the same place as the add new staff button, but when we moved the new project button to the header, the spacing was messed up. I thought the empty div was supposed to help the spacing, but now looking at projects page vs staff page, the spacing still doesnt match 😅

Reply:
ah, that makes sense 😅

Reply:
I agree with Chia, and I think if we're going to make this change to Projects, we should make sure we're consistent and apply this to the Staff page as well.